### PR TITLE
Update_documentation

### LINKED ISSUE #485: Add example in documentation for multiple runs of the same sample
### Description of feature

https://nf-co.re/taxprofiler/1.1.7/docs/usage#multiple-runs-of-the-same-sample

Would be good to add an example of lanes in documentation above.

### PR DESCRIPTION
**Pull request by sofstam**
Closes #485 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -46,11 +46,13 @@ The `sample` identifiers have to be the same when you have re-sequenced the same
...
 sample,run_accession,instrument_platform,fastq_1,fastq_2,fasta
-2612,run1,ILLUMINA,2612_run1_R1.fq.gz,,
-2612,run2,ILLUMINA,2612_run2_R1.fq.gz,,
-2612,run3,ILLUMINA,2612_run3_R1.fq.gz,2612_run3_R2.fq.gz,
+2612,lane1,ILLUMINA,2612_lane1_R1.fq.gz,ILLUMINA,2612_lane1_R2.fq.gz,
+2612,lane2,ILLUMINA,2612_lane2_R1.fq.gz,ILLUMINA,2612_lane2_R2.fq.gz,
+2612,lane3,ILLUMINA,2612_lane3_R1.fq.gz,,
 ```
 
+Please note that the column name `run_accession` is following the ENA terms and is a single or paired-end set of demultiplexed FASTQs. Given that demultiplexing happens per lane, each sequencing pair from each lane is a 'run' and therefore you get multiple 'runs' per sample, which can span across both lanes and sequencing libraries.
```
Comment (jfy133) [inline docs/usage.md:54]:
```suggestion
::: info
Please note that the column name `run_accession` follows the definition of an ENA 'run'.
A 'run' corresponds to a single or paired-end set of demultiplexed FASTQs. 
Given that demultiplexing of a given library happens per lane, each sequencing pair from each lane is a 'run'.
Therefore, for each sample, you may get multiple 'runs' consisting of _both_ lanes (of the same library) _and_ sequencing libraries. 
Therefore ensure that each `run_accession` ID is unique, even if from the same sample!
:::
```

Comment (jfy133) [reply docs/usage.md:54]: What do you think @sofstam ? I tried to be more explicit

**Review by jfy133 — COMMENTED** [review]
Rephrase suggestion

File: docs/usage.md (hunk 1 above)
Comment (sofstam) [reply docs/usage.md:54]: 🚀 Very detailed!

**Review by jfy133 — APPROVED** [review]

**Pull request merged**